Taxi/Taxi/Presenter/CalendarView.swift

```diff
@@ -0,0 +1,175 @@
+//
+//  CalendarView.swift
+//  Taxi
+//
+//  Created by sanghyo on 2022/06/09.
+//
+
+import SwiftUI
+
+struct CalendarView: View {
+    @State private var currentDate = Date()
+    @State private var selectedDate = Date()
+    @State private var currentMonth = 0
+    private let today = Date()
+    private let taxiParties: [TaxiParty] = TaxiPartyMockData.mockData
+    private let action: () -> Void
+    let days = ["일", "월", "화", "수", "목", "금", "토"]
+    let intDateConverter = IntDateConverter()
+
+    init(action: @escaping () -> Void) {
+        self.action = action
+    }
+
+    var body: some View {
+        VStack {
+            VStack(spacing: 20) {
+                navigation
+                dayOfWeek
+            }
+            datePicker
+        }
+        .frame(maxWidth: .infinity)
```

Comment:
이미 내부에 dayOfWeek이나 dayCell이 .infinity 속성이기 때문에 이 전체 VStack도 최대 너비를 차지하게 됩니다.
따라서 여기서 또 .infinity를 줄 필요는 없습니다.
전체 컨텐츠에는 좌우 마진값을 주는게 좋습니다.
```suggestion
        .padding(.horizontal)
```

Reply:
이 부분에서 궁금한게 있는데, 달력을 컴포넌트용으로 만들었으니 사용하는 사람들이 padding()을 적절하게 사용하면 된다고 생각하여 padding은 아직 넣어주지 않은 상태입니다.
컴포넌트 단계에서 padding을 주는게 좋을까요? 아니면 사용하는 사람이 padding을 적절히 주는게 좋을까요?

Reply:
못넘어가게... 할까요... 괴롭히지 마세요 ㅠㅠㅠ 흑흑 내일 수정해보겠습니다
사실 인터파크 같은 앱에서도 못넘어가게 하기는하더라구요 하핳

Reply:
> 이 부분에서 궁금한게 있는데, 달력을 컴포넌트용으로 만들었으니 사용하는 사람들이 padding()을 적절하게 사용하면 된다고 생각하여 padding은 아직 넣어주지 않은 상태입니다.
> 컴포넌트 단계에서 padding을 주는게 좋을까요? 아니면 사용하는 사람이 padding을 적절히 주는게 좋을까요?

이건 저도 들어보니까 고민이 되네요 ㅎㅎ
컴포넌트에 있는걸 뺄 수는 없지만 없는걸 넣을 수는 있으니가 쓰지 않는게 더 좋을수도?

Reply:
> 못넘어가게... 할까요... 괴롭히지 마세요 ㅠㅠㅠ 흑흑 내일 수정해보겠습니다
> 사실 인터파크 같은 앱에서도 못넘어가게 하기는하더라구요 하핳

😈